tardis_env3.yml

```diff
@@ -10,11 +10,9 @@ dependencies:
   - scipy=1.1
   - pandas=0.24
   - astropy=3
-
   - numba=0.43
   - numexpr
   - Cython=0.29
```

Comment:
I'm not sure about this - if these packages are related somehow (I guess that's why there are newlines), we can add a comment at top to mention the purpose?

Reply:
I think that was the VSCode YAML extension/autoformatter.